package.json

```diff
@@ -14,13 +14,16 @@
   "devDependencies": {
```

Comment:
Can you add in https://www.npmjs.com/package/chromedriver as a `devDependency` instead of requiring developers to install it out-of-band? I know that you were including it in the Travis configuration, but it could be nice to explicitly list it here.

I *think* that would also help make sure that it gets cached in between Travis runs, but I don't really understand Travis caching, so I might be wrong.